examples/v2/project_creation/project.py

```diff
@@ -98,3 +101,35 @@ def GenerateConfig(context):
     })
 
   return {'resources': resources}
+
+def MergeCallingServiceAccountWithOwnerPermissinsIntoBindings(env, properties):
+  """ A helper function that merges the acting service account of the project
+      creator as an owner of the project being created
+  """
+  service_account = ('serviceAccount:{0}@cloudservices.gserviceaccount.com'
+                     .format(env['project_number']))
+  set_creator_sa_as_owner = {
+      'role': 'roles/owner',
+      'members': [
+          service_account,
+      ]
+  }
+  if ('iam-policy' not in properties or
```

Comment:
Should this be "and"? Or would that be too lenient since it would fix a basically invalid iam-policy block?